sintr_analysis_server_example/lib/query/completion_metrics.dart

```diff
@@ -5,30 +5,39 @@
 library sintr_worker_lib.completion;
 
 import 'dart:async';
+import 'dart:convert';
+import 'dart:math';
 
 import 'package:sintr_worker_lib/query.dart';
 import 'package:sintr_worker_lib/session_info.dart';
+import 'package:sintr_worker_lib/instrumentation_transformer.dart';
 
 const AVE = 'ave';
 const INCOMPLETE = 'incomplete';
 const MAX = 'max';
 const MIN = 'min';
+const RESPONSE_TIMES = 'responseTimes';
+const RESPONSE_TIME_CLUSTERS = 'responseTimeClusters';
+const RESULT_COUNTS = 'resultCounts';
+const RESULT_COUNT_CLUSTERS = 'resultCountClusters';
 const TOTAL = 'total';
 const V90TH = '90th';
 const V99TH = '99th';
-const VALUES = 'values';
 const VERSION = 'version';
 
 /// Process log line that was encoded by [_composeExtractionResult]
 /// and return a map of current results
-final completionReducer = (String sdkVersion, int completionTime, Map results) {
+final completionReducer = (String sdkVersion, List extracted, Map results) {
```

Comment:
The standard signature for a reducer is (key, list of values) -> map of results.

Do I understand correctly that 'extracted' here contains a list of completion times?